src/runtime_src/core/edge/user/device_linux.cpp

```diff
@@ -88,6 +88,8 @@ struct devInfo
   get(const xrt_core::device* device,key_type key)
   {
     auto edev = get_edgedev(device);
+    device->get_device_info(&deviceInfo);
```

Comment:
This can be done by calling xclGetDeviceInfo2 directly:
`xclGetDeviceInfo2(device->get_device_handle(), &deviceInfo);`

The ishim interface is only for user space code that doesn't directly link with libxrt_core.  But here we are in the core library itself so we can call anything from libxrt_core directly.

Reply:
did the changes as suggested